internal/blueprint/blueprint.go

```diff
@@ -20,9 +23,34 @@ type Package struct {
 	Version string `json:"version,omitempty"`
 }
 
+type translator interface {
+	translate(b *Blueprint) *pipeline.Pipeline
+}
```

Comment:
This type is called `translator`, but the files are `XXX_output.go`. Does it make sense to give those consistent names?

Reply:
Yeah, I'll rename them all "outputs".